server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java

```diff
@@ -371,7 +377,7 @@ public static RealmRepresentation toRepresentation(RealmModel realm, boolean int
         rep.setAttributes(attributes);
 
         if (!internal) {
-            rep = StripSecretsUtils.strip(rep);
+            return StripSecretsUtils.strip(rep);
```

Comment:
This could cause issues in the future if there are something added later on. This is intentionally not returning the JSON at this point. Please don't change things that are not relevant to PR.